frontend/libs/studio-components/src/components/StudioTextField/StudioIconTextfield/StudioIconTextfield.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Should we add a test to verify that "...rest" works as intended? For example, check that onChange functions as expected and is sent in as a callback? 😊